src/main/jte/bookings/create/success_staged.jte

```diff
@@ -11,7 +11,7 @@
         <div class="max-w-md">
             <h1 class="text-5xl font-bold">${context.localize("bookings.staged.title")}</h1>
             <p class="py-6">${context.localize("bookings.staged.description")}</p>
-            <a class="btn btn-primary" href="/">${context.localize("index.go")}</a>
+            <a class="btn btn-primary" href="/" aria-label="${context.localize("index.go")}" title="${context.localize("index.go")}">${context.localize("index.go")}</a>
```

Comment:
This text is already the text of the button, I don't think repeating it is necessary.